packages/frontend/src/Layout/Footer/Footer.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import {
+  SocialsWrapper,
+  Wrapper,
+  FooterItemsWrapper,
+  FooterItem,
+  Copyright,
+  StyledLink,
+  Text,
+  IconsWrapper,
+} from "./Footer.styled";
+import { Logo } from "../../common/ui/Logo.styled";
+import { Routes } from "src/enums/routes.enum";
+
+import { socials } from "../../components/SocialIcon/config";
+import { SocialIcon } from "../../components/SocialIcon/SocialIcon";
+import { footerItems } from "./config";
+
+//placeholder
+
+export const Footer = () => {
+  return (
+    <Wrapper>
+      <SocialsWrapper>
+        <Logo to={Routes.Home}>Bochen</Logo>
+        <IconsWrapper>
+          {socials.map((el) => {
+            return <SocialIcon key={el.src} color={el.color} src={el.src} link={el.link} />;
```

Comment:
jeżeli nazwiesz propy tak samo jak wartości obiektu SocialsInfo to możesz dać <SocialIcon key={el.src} {...el}/>